opennms-dao/src/test/java/org/opennms/netmgt/dao/support/ObjectNameStorageStrategyTest.java

```diff
@@ -41,6 +41,13 @@
  */
 public class ObjectNameStorageStrategyTest {
 
+    /**
+     * Copied from GenericTypeResource
+     */
+    private static String sanitizeInstance(String instance) {
```

Comment:
Can we make the one we added to the `MockCollectionResource` accessible and use that one instead of adding it again here?

Reply:
Sure!